documentation/Extension_Point_Content_Model.md

```diff
@@ -0,0 +1,51 @@
+# Content Model
+
+Dynamic Extensions provides a convenient way of registering custom content models with the Alfresco Data Dictionary. (See the <a href="http://wiki.alfresco.com/wiki/Data_Dictionary_Guide">Data Dictionary Guide</a> for more info on content models.)
```

Comment:
This file uses HTML rather than markdown? It renders fine on Github, so it's not really a problem, just a bit strange.